kt/plugins/godot-intellij-plugin/src/main/kotlin/godot/intellij/plugin/gradle/GodotKotlinJvmSettings.kt

```diff
@@ -13,6 +13,7 @@ import com.intellij.ui.EditorNotifications
 import godot.intellij.plugin.GodotPluginBundle
 import godot.intellij.plugin.notification.SettingsFetchingNotification
 import godot.plugins.common.GodotKotlinJvmPropertiesFile
+import godot.plugins.common.gradle.action.FetchProjectModelsBuildAction
```

Comment:
why is this added ?

Reply:
because we moved that class from the same package to the common module instead. so an import is needed